webapi/src/main/scala/org/knora/webapi/responders/v1/PermissionsResponderV1.scala

```diff
@@ -737,23 +735,23 @@ class PermissionsResponderV1 extends ResponderV1 {
       * The permissions include any default object access permissions defined for the resource class and on any groups the
       * user is member of.
       *
-      * @param projectIRI the IRI of the project.
-      * @param resourceClassIRI the IRI of the resource class for which the default object access permissions are requested.
-      * @param propertyIRI the IRI of the property for which the default object access permissions are requested.
+      * @param projectIri the IRI of the project.
+      * @param resourceClassIri the IRI of the resource class for which the default object access permissions are requested.
+      * @param propertyIri the IRI of the property for which the default object access permissions are requested.
       * @param permissionData the permission data of the user for which the default object access permissions are requested.
       * @return an optional string with object access permission statements
       */
-    def defaultObjectAccessPermissionsStringForEntityGetV1(projectIRI: IRI, resourceClassIRI: Option[IRI], propertyIRI: Option[IRI], permissionData: PermissionDataV1): Future[Option[String]] = {
+    def defaultObjectAccessPermissionsStringForEntityGetV1(projectIri: IRI, resourceClassIri: Option[IRI], propertyIri: Option[IRI], permissionData: PermissionDataV1): Future[DefaultObjectAccessPermissionsStringResponseV1] = {
```

Comment:
Do we need a special return message to just return a string?

Also, I have an informal convention that a request called ``SomethingSomethingGetRequestV1`` is returning a ``SomethingSomethingGetResponseV1``, and that ``SomethingSomethingGetV1`` is returning something else, in this case a ``String``. Before you ask, the whole distinction between ``SomethingSomethingGetRequestV1`` and ``SomethingSomethingGetV1`` is needed because you would like ``NotFound`` exceptions to be thrown inside the responder methods called from routes, which on the other hand makes them unsuitable for internal use.

So, I would suggest that we either keep the distinction between ``SomethingSomethingGetRequestV1 `` and ``SomethingSomethingGetV1 ``, so then ``DefaultObjectAccessPermissionsStringResponseV1`` would need to be renamed in something like ``DefaultObjectAccessPermissionsStringV1``, or we move the throwing of ``NotFound`` error messages to the routes, which would basically entail writing a small match statement.